GWAS/PCA.ipynb

```diff
@@ -1182,8 +1175,8 @@
     "              scale_color_manual(values=color_list) +\n",
     "              theme_classic()\n",
     "        }}\n",
-    "    unit = 4\n",
-    "    n_col = min(4, k)\n",
+    "    unit = 2\n",
```

Comment:
@UxxUnet What's the issue with displaying 4 images per row?

Reply:
If we are trying to decide between 2 or 4 why don't we make it an option? 

```
parameter: n_col = 4
R: ...
    ...
    n_col = min(${n_col}, k)

```

does it make sense?